pkg/netutil/netutil.go

```diff
@@ -44,6 +44,10 @@ func NewCNIEnv(cniPath, cniConfPath string) (*CNIEnv, error) {
 		NetconfPath: cniConfPath,
 	}
 	var err error
+	err = e.ensureDefaultNetworkConfig()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
We have to make sure to lock the CNI dir (`git grep pkg/lockutil`)

Reply:
Seems to already have a lock when create/remove network. 
https://github.com/containerd/nerdctl/blob/master/cmd/nerdctl/network_create.go#L149

`ensureDefaultNetworkConfig` just makes sure the default network exists.